tests/toolkit/matl_buy_policy_tests.cc

```diff
@@ -42,71 +42,125 @@ class MatlBuyPolicyTests: public ::testing::Test {
 };
 
 TEST_F(MatlBuyPolicyTests, Init) {
-  double cap = 5;
+  double cap = 10;
```

Comment:
No, not anymore. I was previously adding all the sS and RQ tests into Init, but realized that 1. re-initializing a buy policy with a different buying strategy leaves the old parameters hanging around and probably isn't good practice so I separated the tests and 2. refactored the test after making that change so the cap isn't really important anymore. I'll roll it back since there's no reason for it any more